Source/WebKit/UIProcess/WebPageProxyTesting.cpp

```diff
@@ -210,4 +210,16 @@ Ref<WebPageProxy> WebPageProxyTesting::protectedPage() const
     return m_page.get();
 }
 
+void WebPageProxyTesting::resetStateBetweenTests()
+{
+    m_page->protectedLegacyMainFrameProcess()->resetState();
```

Comment:
Need to call `protectedPage()` instead since `protectedLegacyMainFrameProcess()` is an out of line function.
Maybe you can define a local variable since you're calling protectedPage() in line 220 as well:
```
RefPtr page = m_page.get();
```

Reply:
Thanks!  I also need to investigate the assertions in the site isolation layout tests.